skel/bq/Makefile

```diff
@@ -0,0 +1,77 @@
+# Programs
+SED ?= sed
+GREP ?= grep
```

Comment:
This file is not using GREP. It's not really disturbing though, so contributors know that they would have to use this notation.